### ISSUE TITLE
Classification with no segmentation

### ISSUE THREAD

**Issue by sharadxkumar** [NaturalIntelligence/imglab#128]
I just started using imglab and thanks for providing such a useful tool.  I was wondering if its possible to just classify an image without having to segment it first with a rectangle(or any other shape)?   For example, I just want to classify the image with following classes (dog, cat, sheep, others) without having to draw a rectangle around the animal.  Is that possible in the current version of the tool?

**Comment by amitguptagwl** [comment]
Yes, imglab internally supports it. However, UI is not updated to take that input. Also mentioned in #86 

> When no shape or point is selected then label panel should display information about image. Which should be updated when the image is changed from thumbnail slider ...

If you're a developer, I'll suggest you to rais a PR. I can guide you where to do the changes.
